### PR TITLE
Allow to configure whether to gather global variables and statuses in input/mysql

### PR DESCRIPTION
**Pull request by xert**
### Required for all PRs:
- [x] CHANGELOG.md updated
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by danielnelson** [comment]
@xert Can you rebase due to changes to the mysql plugin?

**Comment by glinton** [comment]
bump @xert, also be aware of #4306 as it may add more conflicts.

File: CHANGELOG.md [hunk 1]
```diff
@@ -139,6 +139,7 @@ consistent with the behavior of `collection_jitter`.
 - [#1600](https://github.com/influxdata/telegraf/issues/1600): Fix quoting with text values in postgresql_extensible plugin.
 - [#1425](https://github.com/influxdata/telegraf/issues/1425): Fix win_perf_counter "index out of range" panic.
 - [#1634](https://github.com/influxdata/telegraf/issues/1634): Fix ntpq panic when field is missing.
+- [#1642](https://github.com/influxdata/telegraf/pull/1642): input/mysql: Allow to configure whether to gather global variables and statuses
```
Comment (glinton) [inline CHANGELOG.md]: Remove this as we update the changelog when we merge in order to prevent merge conflicts.

**Comment by russorat** [comment]
Closing this due to inactivity. If you are interested in this, please comment.

**Pull request closed without merging**